packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -117,7 +117,7 @@ const TutorialStore = types
     function createWorkflowObserver () {
       const workflowDisposer = autorun(() => {
         const workflow = tryReference(() => getRoot(self).workflows.active)
-        if (!self.loaded && workflow) {
+        if (workflow) {
           self.reset()
```

Comment:
Not a blocker, but this will break when the workflow and tutorial are loaded from a snapshot. I’ll continue working on that.

Reply:
I can continue working on that in #2675.